### ISSUE TITLE
STEP 3 Error...

### ISSUE THREAD

**Issue by dustycy** [streamlink/streamlink-portable#25]
Received the following STEP 3 error message while trying to build a portable EXE for the latest stable streamlink-0.14.0 release:

System.IO.IOException: Access to the path 'C:\Users\LAK\Downloads\streamlink-portable-master\Streamlink for Windows (Compiled)\Files\TEMP\streamlink-0.14.0' is denied.
   at System.IO.Directory.InternalMove(String sourceDirName, String destDirName, Boolean checkHost)
   at Microsoft.VisualBasic.FileIO.FileSystem.RenameDirectory(String directory, String newName)
   at Microsoft.VisualBasic.MyServices.FileSystemProxy.RenameDirectory(String directory, String newName)
   at Streamlink_for_Windows.MainWindow.BW_STEPS_DoWork(Object sender, DoWorkEventArgs e)

I am using VERSION "Build date: 06/09/2018 (MM/DD/YYYY)".

**Comment by RosadinTV** [comment]
I can't reproduce this bug.
Maybe a program is blocking the folder (like antivirus software).
You could also try restarting your PC or running the program with administrator rights.
![streamlinkportable014test](https://user-images.githubusercontent.com/13382316/42009686-dde26d34-7a61-11e8-9c6c-0b9da3f40573.png)
PS: Soon i'll upload a precompiled release of 0.14.0 [here](https://github.com/streamlink/streamlink-portable/releases) (as usual).

**Comment by dustycy** [comment]
You are correct.

Avast Free Antivirus have been locking certain PDF files that I was not able to delete, rename or move them. (They are aware of the issue and will release an update next week.)

I have uninstalled Avast and installed Kaspersky Free Antivirus. It is Kaspersky blocking your program. I paused Kaspersky and have created a portable EXE for the latest stable release.

Sorry about that. Thanks.
